CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ elseif(OS_POSIX)
   if(OS_LINUX)
     option(ENABLE_WAYLAND "Enable building with support for Wayland (Linux)" ON)
     option(BUILD_FOR_PPA "Build for PPA distribution" OFF)
+    option(ENABLE_WHATSNEW_LINUX "Enable WhatsNew dialog on Linux" OFF)
```

Comment:
I don't think we need an additional flag for this - make `ENABLE_WHATSNEW` an `INTERNAL` cache variable (set to "ON") on macOS and Windows, make it an `option` on `OS_LINUX`, also defaulting to "ON".

That would make the changes above (explicitly enabling it in the build script and for FlatPak) unnecessary and forces other package maintainers to explicitly turn it off (as desired for Linux).

Reply:
Where would be the most appropiate place to set the internal cache variable? `ObsDefaults_{Windows,macOS}` or just at the top of this file?